src/pages/DataAccessRequestApplication.js

```diff
@@ -247,19 +179,16 @@ class DataAccessRequestApplication extends Component {
       formData.zipcode = rpProperties.zipcode != null ? rpProperties.zipcode : '';
       formData.country = rpProperties.country != null ? rpProperties.country : '';
       formData.state = rpProperties.state != null ? rpProperties.state : '';
-      formData.piName = rpProperties.piName !== null ? rpProperties.piName : '' ;
+      formData.piName = rpProperties.piName !== null ? rpProperties.piName : '';
```

Comment:
I think this can be improved with `get(rpProperties, "piName", "");` from lodash.

Reply:
Good call. Most of this was copy/paste from the other components ... I can clean it up some more.

Reply:
Ah ... re-reading this ... this was a lint checking change. I prefer to leave as much of the old code in place if it isn't directly related to the era commons stuff.

Reply:
It makes sense.